### DISCUSSION TITLE
Gnome doesnt close main window

### DISCUSSION THREAD

**Discussion by duqkefly** [hluk/CopyQ#2023] [Ideas]
Why never close main window in gnome.?? Its full work on cinnamon but not equal on gnome..

**Comment by hluk** [comment]
Try enabling "Hide main window" in Layout tab in Preferences. It is disabled by default so users can access the main window easily even if they do not have tray (in such case window is minimized instead instead of closed/hidden).

**Comment by duqkefly** [comment]
Thanks, i will try..
